.gitignore

```diff
@@ -30,9 +25,9 @@ venv
 .cache/
 coverage/
 
-# For docker
+# Docker
 .docker-build
-docker/config/webapp.env
+docker/config/webapp.env*
```

Comment:
I believe that one is incorrect. We have a `webapp.env.template` that we want to track in git. Why did you change this rule?

Reply:
`docker/config/webapp.env-e` popped up for me.

Reply:
Huh, OK, that's surprising. Definitely don't see that locally. Can you add specifically `-e` then?

Reply:
@adngdb Should I instead list this specific file?

Reply:
Yeah that's what I meant.